background_scripts/commands.coffee

```diff
@@ -97,7 +97,7 @@ Commands =
     # Keys are either literal characters, or "named" - for example <a-b> (alt+b), <left> (left arrow) or <f12>
     # This regular expression captures two groups: the first is a named key, the second is the remainder of
     # the string.
-    namedKeyRegex = /^(<(?:[amc]-.|(?:[amc]-)?[a-z0-9]{2,5})>)(.*)$/
+    namedKeyRegex = /^(<(?:[amc]-.|(?:[amc]-)?[a-z0-9]{2,})>)(.*)$/
```

Comment:
They're used for different reasons:
* the one on line 44 is to make non-printable key names (length >= 2) lower-case, and only matches at most one status keys (`altKey/metaKey/ctrlKey`) and a non-printable key,
which means shiftKey is not supported.
* the one on line 100 is to split all key names which is wrapped by `<` and `>`.

In fact, I think we should support `shiftKey` for non-printable keys and combination of status keys, you may see #2210 for what I want.